### ISSUE TITLE
Section 3.7 (culture): the proof points and evaluations don't seem to address the core of the issue.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/wcag-maturity-model#50]
It seems to me that the proof points and ratings in section 3.7 are all indicative of the quality of an organization's culture with respect to accessibility, but that they don't address core issues one would reasonably consider important, such as:

To what extent are issues and concerns raised by employees, clients etc., with disabilities actually paid attention to and addressed through concrete actions?

To what degree is the expertise of accessibility specialists or others who have accessibility-related knowledge within the organization not only respected by colleagues, but acted upon in the decisions which are made about the design, development and procurement of ICT systems?

How consistently is a collaborative approach taken in which all parties seek to ensure accessibility-related needs are met?

On the negative side, if there is a culture of ignoring, disrespecting, or not acting on the advice of people with accessibility expertise, then the organization doesn't have an inclusive culture - even if other indicators are present. Likewise if the needs of people with disabilities are frequently sidelined, ignored, or not treated as priorities (whether those people are internal or external to the organization).

In future drafts of this section, I think it would be helpful to give closer attention to the ways in which culture is manifest in concrete behaviours (i.e., reflected in how people make decisions and treat each other in practice) - especially in contexts that affect accessibility-related outcomes.

**Comment by Helixopp** [comment]
Thank you for your comment. You make important points. However, we believe that the issues you identified are inherent in the proof points, without the document dictating that level of detail.
